CleverPush/Source/CleverPushInstance.m

```diff
@@ -182,6 +182,7 @@ - (void)setTrackingConsent:(BOOL)consent {
     if (hasTrackingConsent) {
         [self fireTrackingConsentListeners];
     } else {
+        [self removeSubscriptionTagsAndAttributes];
```

Comment:
Only call this when hasTrackingConsent was true before this method was called. So we need to compare the previous value before overriding it here https://github.com/cleverpush/cleverpush-ios-sdk/pull/178/files#diff-f5326f84792597a7145e9b2f5be8823e3bc651f5b4faac02dc05dca951eee90fR180